src/Symfony/Component/Form/Tests/AbstractLayoutTest.php

```diff
@@ -2668,6 +2669,42 @@ public function testHelpWithTranslationParameters()
         );
     }
 
+    /**
+     * @requires function Symfony\Component\Translation\TranslatableMessage::__construct
```

Comment:
This doesn't look needed.

Reply:
indeed. found it elsewhere but will remove it.